src/infra/error-handling/index.ts

```diff
@@ -47,15 +45,10 @@ export function captureException(
 
 	Sentry.withScope((scope) => {
 		if (options != null) {
-			const { tags, req, extra } = options;
+			const { tags, extra } = options;
 			if (tags != null) {
 				scope.setTags(tags);
 			}
-			if (req != null) {
-				scope.addEventProcessor((evt) =>
-					Sentry.addRequestDataToEvent(evt, req),
```

Comment:
My understanding is that nowadays this should be automatically handled by https://docs.sentry.io/platforms/javascript/guides/node/configuration/integrations/requestdata/ and we should not need to manually do it